Stratis.Bitcoin/Features/BlockStore/readme_blockstoreloop.md

```diff
@@ -0,0 +1,105 @@
+BlockStoreLoop
```

Comment:
You shouldn't need to write file like this manually - it only contains information that you already have in XMLDOC. It could be automatically generated from the XMLDOC providing the formatting is done correctly.